<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>I.A.D.M.</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="theCollectionName"/>
      </item>
      <item>
       <widget class="QPushButton" name="theDlMetadataButton">
        <property name="text">
         <string>Download Metadata</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="theMDTable"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Space Required:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="theSpaceReqd"/>
      </item>
      <item>
       <widget class="QPushButton" name="theStartDlFilesButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start Downloading!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>File:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="theCurrentFileDlName"/>
      </item>
      <item row="0" column="2">
       <widget class="QProgressBar" name="theFileProgress">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Total:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="theBytesDownloaded"/>
      </item>
      <item row="1" column="2">
       <widget class="QProgressBar" name="theTotalProgress">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="theConfigureAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="theAboutToolAction"/>
    <addaction name="theAboutQtAction"/>
   </widget>
   <addaction name="menuConfigure"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="theAboutToolAction">
   <property name="text">
    <string>About I.A.D.M.</string>
   </property>
  </action>
  <action name="theConfigureAction">
   <property name="text">
    <string>Configure...</string>
   </property>
  </action>
  <action name="theAboutQtAction">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
